src/components/common/ClientIntakeForm/IntakeForms/DocumentUpload.js

```diff
@@ -0,0 +1,244 @@
+      key: 'type',
+      filters: documentTypeFilter,
+    },
+    {
+      title: 'Options',
+      key: 'options',
+      render: (text, record, i) => (
+        <Space size="middle">
+          <Button
+            icon={<DeleteOutlined />}
+            onClick={e => handleDelete(record)}
+          ></Button>
+        </Space>
+      ),
+    },
+    {
+      title: 'Uploaded',
+      key: 'uploaded',
+      render: (text, record, i) => (
+        <Space size="middle">
+          {record.success ? (
+            <CheckCircleOutlined />
+          ) : (
+            <ExclamationCircleOutlined />
+          )}
+        </Space>
+      ),
+    },
+  ];
+
+  useEffect(() => {
+    if (props.client.id !== 0) {
+      axios
+        .get(`http://localhost:8000/api/documents/${props.client.id}`)
+        .then(documents => {
+          console.log(documents.data);
+          props.setDocuments(documents.data);
+          console.log(documents.data);
+        })
+        .catch(error => {
+          alert('ERROR ' + error);
+        });
+    }
+  }, [props.client]);
+
+  const handleDelete = file => {
+    axios
+      .post(`http://localhost:8000/api/documents/${props.client.id}`, file)
+      .then(() => {
+        props.deleteDocument(file);
+      })
+      .catch(error => {
+        alert('ERROR ' + error);
+      });
+  };
+
+  const upload = async e => {
+    console.log(e.file.name, { ...e.file });
+
+    setDisabled(true);
+    let file = e.file;
+    let fileParts = e.file.name.split('.');
+    let fileName = fileParts[0];
+    let fileType = fileParts[1];
+    axios
+      .post('http://localhost:8000/api/documents/sign_s3', {
+        fileName: fileName,
+        fileType: fileType,
+      })
+      .then(response => {
+        var returnData = response.data.data.returnData;
+        var signedRequest = returnData.signedRequest;
+        var url = returnData.url;
+        var options = {
+          headers: {
+            'Content-Type': fileType,
+          },
+        };
+        axios
+          .put(signedRequest, file, options)
+          .then(result => {
+            const documentObj = {
+              name: fileName,
+              success: true,
+              url: url,
+              documentType: documentType,
+              extension: fileType,
+            };
+            props.addDocument([documentType, documentObj]);
+            axios
+              .post(
+                `http://localhost:8000/api/documents/${props.client.id}/update`,
+                { [documentType]: documentObj }
+              )
+              .then(result => {
+                setDocumentType(null);
+              })
+              .catch(error => {
+                alert('ERROR ' + error);
+              });
+          })
+          .catch(error => {
+            alert('ERROR ' + error);
+          });
+      })
+      .catch(error => {
+        alert('ERROR ' + error);
+      });
+  };
+
+  const onSelect = e => {
```

Comment:
Do you think this function would be more clear to other devs if it's name was specific to the DOM element and event?